www/stats_operations.py

```diff
@@ -1,34 +1,50 @@
 import csv
 import time
+from datetime import datetime
 from .db import Stats
 from io import StringIO
 from flask import abort
 
 def read_stats():
     csv_output = StringIO()
 
-    writer = csv.DictWriter(csv_output, Stats._meta.fields.keys())
+    keys = Stats._meta.fields.keys();
+
+    new_keys = list(keys)
+    new_keys.remove('id')
+
+    writer = csv.DictWriter(csv_output, new_keys)
     writer.writeheader()
+    results = Stats.select().dicts()
+
+    for result in results:
+        result.update({
+            'already_existed' : 'true' if bool(result.get('Stats.already_existed')) else 'false',
+            'timestamp' : result.get('timestamp').isoformat()
+                       })
+
+        result.pop('id')
```

Comment:
Falls möglich, schon beim `Stats.select` oben handhaben. (Vermutlich können diesem `new_keys` als die gewünschten Spalten mitgegeben werden, evtl. müssen sie dazu in einzelne Argumente entpackt werden, was mit `*` davor gehen sollte, also
```suggestion
    results = Stats.select(*new_keys).dicts()

    for result in results:
        result.update({
            'already_existed' : 'true' if bool(result.get('Stats.already_existed')) else 'false',
            'timestamp' : result.get('timestamp').isoformat()
                       })
```
)

Reply:
Geht leider nicht. Das Select Statement kann leider nicht mit strings gefüttert werden.

Reply:
Alternative wurde mit https://github.com/geometalab/cf_audit/commit/63225933ac274a8a20cdc6230068a21dbb7014d9 umgesetzt.